.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+fake_test/
```

Comment:
revert.

don't put test directories on your git project.

it is ok to add test files/resources, just put a good readme and a easy way to execute then.

```suggestion
```